ide/dune

```diff
@@ -15,7 +16,13 @@
  (name coqide_main)
  (public_name coqide)
  (package coqide)
- (modules (:standard \ idetop))
- (libraries threads str lablgtk2.sourceview2 coq.lib coqide.protocol))
+ (modules coqide_main)
+ (libraries coqide.core))
 
-(ocamllex utf8_convert config_lexer coq_lex)
+(executable
+ (name idetop)
+ (public_name coqidetop.opt)
+ (package coqide)
+ (modules idetop)
+ (libraries coq.toplevel coqide.protocol)
```

Comment:
Why does `coqide_main` depends on the library `coqide.core` but `idetop` does not?

Reply:
`coqide.core` contains all the "client-side" graphical routines, `idetop` is just `coqtop` with some extra capabilites, acting as the server. The only common part they have is the protocol, which is gtk-free.

Reply:
`coqide.core` is also said to contain module `idetop` though.

Reply:
the line
```
(:standard \ idetop coqide_main)
```
means "all modules but `idetop` and `coqide_main`, see https://dune.readthedocs.io/en/latest/dune-files.html#ordered-set-language